pkg/packagekit/updates.css

```diff
@@ -38,8 +38,11 @@ tr.security {
 }
 
 /* don't let the install progress bar get too wide */
-.progress {
+.progress-main-view {
     max-width: 60rem;
+    margin-top: 10ex;
+    margin-left: auto;
+    margin-right: auto;
```

Comment:
This is nice indeed! I tested and updated that locally; I pushed that, and also added a Closes # while I'm at it.